Python/Cryptocurrency_Converter/crypto_converter.py

```diff
@@ -1,110 +1,134 @@
+
         if user_inp == "1":
-            result = convert_crypto_to_currency(
-                amount, cryptocurrency, currency)
-            print(f"{amount} {cryptocurrency} in {currency} is {result}")
+            crypto_type = input("\nEnter the Cryptocurrency Type  :  ")
+            fiat_type = input("Enter the Fiat Money Type  :  ")
+            try: crypto_value = float(input("\nEnter the Cryptocurrency Value  :  "))
+            except:
+                print("Bad Input!\nPlease Enter \"y\" if you wish to continue, else \"n\".")
+                choice = input()
+                if choice == "y": continue
+                elif choice == "n": return
+                else:
+                    print("Bad Input Again!")
+                    return
+            result = convert_crypto_to_fiat(crypto_value, crypto_type.upper(), fiat_type.upper())
+            if result is None: return
+            elif result == 1: continue
+            print(f"{crypto_value} {crypto_type.upper()} in {fiat_type.upper()} is  -->  {result}")
+
         elif user_inp == "2":
-            result = convert_crypto_to_currency(
-                amount, cryptocurrency, currency)
-            print(f"{amount} {currency} in {cryptocurrency} is {result}")
-        elif user_inp == "Q":
-            break
+            fiat_type = input("\nEnter the Fiat Money Type  :  ")
+            crypto_type = input("Enter the Cryptocurrency Type  :  ")
+            try: fiat_value = float(input("\nEnter the Fiat Money Value  :  "))
+            except:
+                print("You did not enter a number!\nPlease Enter \"y\" if you wish to continue, else \"n\".")
+                choice = input()
+                if choice == "y": continue
+                elif choice == "n": return
+                else:
+                    print("Bad Input Again!")
+                    return
+            result = convert_fiat_to_crypto(fiat_value, fiat_type.upper(), crypto_type.upper())
+            if result is None: return
+            elif result == 1: continue
+            print(f"{fiat_value} {fiat_type.upper()} in {crypto_type.upper()} is  -->  {result}")
+        
+        elif user_inp == "Q" or user_inp == "q": return
+
         else:
-            print("Wrong Input! try again")
+            print("\nYour Input was NOT in the choices. Try again later.\n")
+            return
+
+
+if __name__ == "__main__":
+    """The driver code."""
+    info_msg = """
+We convert currency -> Cryptocurrency to Fiat Money and vice-versa.\n\n
+Possible Cryptocurrencies available for conversion are : 
+\t BTC  :  Bitcoin
+\t ETH  :  Ethereum.
+\t ETC  :  Ethereum Classic
+\t BCH  :  Bitcoin Cash
+\t LTC  :  Litecoin
+\t ZEC  :  Zcash
+\t ZRX  :  Ox\n\n
+Press (1)  :  To convert Cryptocurrency to Fiat Money.
+Press (2)  :  To convert Fiat Money to Cryptocurrency.
+Press (Q/q)  :  If you're wanting to Quit
+"""
+    print(info_msg)
+    stat = main()
+    if stat is not None:
+        print(stat)
```

Comment:
Please add a EOF

Reply:
Hey @vybhav72954 !
I'm afraid I don't know what exactly that means. Can you please guide me through?

Reply:
EOF is just a blank line at the end of the file. XD

Reply:
I guess that shouldn't take long!
Will address the change asap. Thanks @vybhav72954 .